src/lib/svg_text_utils.js

```diff
@@ -459,7 +461,7 @@ function buildSVGText(containerNode, str) {
                         var dummyAnchor = document.createElement('a');
                         dummyAnchor.href = href;
                         if(PROTOCOLS.indexOf(dummyAnchor.protocol) !== -1) {
-                            nodeSpec.href = href;
+                            nodeSpec.href = encodeURI(href);
```

Comment:
FWIW I couldn't identify any vulnerability due to *not* calling `encodeURI` here - quotes and other special characters embedded in `href` still can't break you out of the `href` attribute (while constructing the `onclick` string or otherwise) to where you can cause more damage - but it makes strange items look more url-like.